prow/config/config.go

```diff
@@ -1116,7 +1142,7 @@ func (d *Deck) Validate() error {
 		return fmt.Errorf("deck.skip_storage_path_validation is enabled despite deck.additional_allowed_buckets being configured: %v", d.AdditionalAllowedBuckets)
 	}
 
-	// Note: The RerunAuthConfigs logic isn't deprecated, only the above RerunAuthConfig stuff is
+	// Note: The RerunAuthConfigs logic is deprecated
 	if d.RerunAuthConfigs != nil {
```

Comment:
Would be better to call the `Finalize` function converting `RerunAuthConfigs` to `DefaultRerunAuthConfigs` then validate only once

Reply:
It now validates once! While making this change, I also removed generating a `RerunAuthConfig` in the `FinalizeDefaultRerunAuthConfigs` function, as it is not a functionality we are seeking.

Reply:
nice